### PR TITLE
feat: responsive image and circle and square image

### PR DESCRIPTION
**Pull request by TheAmlanSaikia**
Added Responsive image and square and circle images of fixed dimension
Responsive Images 2

### CODE REVIEW

File: Responsive Images Component/Responsiveimages.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (ianuragsharma) [inline Responsive Images Component/Responsiveimages.html:6]: suggestion :you can add author meta tag

**Review by ianuragsharma — APPROVED** [review]
Looks good

File: Responsive Images Component/Responsiveimages.html (hunk 1 above)
Comment (TheAmlanSaikia) [reply Responsive Images Component/Responsiveimages.html:6]: Okay, thanks will implement it.

**Review by laxmikanthkonthum — APPROVED** [review]
Looks good

**Pull request merged**